andino_navigation/README.md

```diff
@@ -16,32 +16,31 @@ We rely on [Nav2](https://github.com/ros-planning/navigation2) stack in order to
 2. We need a previously recorded map in order to navigate on.
    Refer to [andino_slam](../andino_slam/README.md) to learn how to record a map with Andino.
 
-### Run nav stack
+3. Run Nav Stack (see instructions in [Run nav stack](#run-nav-stack)).
 
-```sh
-ros2 launch andino_navigation bringup.launch.py map:=<path-to-my-map-yaml-file>
-```
+## Simulation
```

Comment:
I think we want to remove the simulation section as we are not having simulation references within this package anymore. Other parts of the readme should be organized such that it makes sense with the title and content after this change.

Reply:
Yes but we also don't have references to the real robot in this package, this part is only an explanation on how to use this package with any simulation (same way as it was explained how to use it with the `andino_bringup` package).

Reply:
My point is that the `## Simulation` section is empty. Isn't it? If so, we should remove it.

Reply:
@agalbachicar It is not empty, it has the `Prerequisites` to be able to launch the Nav 2 stack and use it, same as the `Real Robot` part.

![image](https://github.com/Ekumen-OS/andino/assets/79662866/051ed788-0240-4802-976e-341bc312895d)

Reply:
Ohh I see the full readme in your fork now. Let me clarify then:

- It is expected that no difference is found when running in simulation or in the real robot.
- The navigation stack expects (pre-requisites):
  - A robot (simulated or real) to up and running. When I say a robot I mean a set of publishers, subscribers and services.
  - A map.

So I would expect the following changes:

- Change the text structure so you don't repeat how to obtain a map.
- Make it super explicit that having either a simulated or real robot is a runtime dependency. Refer to those packages and launchfiles as examples to satisfy the dependencies.
- Remove the reference to the Run nav stack and just proceed to it as the natural way of reading an instruction list.

Note you did that, but the structure should be pedantic, not polite. It's important to instruct people to do things in a certain way with imperative language to avoid double interpretations. Look at the following:

Original text:
<pre>
## Real robot

### Prerequisites
1. Andino robot needs to be up and running:

```
ros2 launch andino_bringup andino_robot.launch.py
``` 

2. We need a previously recorded map in order to navigate on. Refer to [andino_slam](https://github.com/JesusSilvaUtrera/andino/blob/jesus/%23228_simulation_dependency/andino_slam/README.md) to learn how to record a map with Andino.

3. Run Nav 2 Stack (see instructions in [Run nav stack](https://github.com/JesusSilvaUtrera/andino/tree/jesus/%23228_simulation_dependency/andino_navigation#run-nav-stack)).

## Simulation

### Prerequisites
1. Andino simulation needs to be up and running. You can use several simulators, for example Gazebo Classic:
```
ros2 launch andino_gz_classic andino_one_robot.launch.py
``` 

2. We need a previously recorded map in order to navigate on. Refer to [andino_slam](https://github.com/JesusSilvaUtrera/andino/blob/jesus/%23228_simulation_dependency/andino_slam/README.md) to learn how to record a map with Andino.

3. Run Nav 2 Stack (see instructions in [Run nav stack](https://github.com/JesusSilvaUtrera/andino/tree/jesus/%23228_simulation_dependency/andino_navigation#run-nav-stack)).
</pre>

Suggested text:
<pre>

## Prerequisites
1. Run the mobility stack in a real Andino robot  or a simulated one:

_Real robot_
```
ros2 launch andino_bringup andino_robot.launch.py
``` 

_Example with Gazebo Classic_
```
ros2 launch andino_gz_classic andino_one_robot.launch.py
``` 

2. Provide a recorded map. Refer to [andino_slam](https://github.com/JesusSilvaUtrera/andino/blob/jesus/%23228_simulation_dependency/andino_slam/README.md) to learn how to record a map with Andino.

</pre>